Dockerfile

```diff
@@ -13,26 +13,32 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.10.3 as builder
+FROM golang:1.13.7 as builder
```

Comment:
maybe when push again update this to 1.13.8
if you are not pushing we can update in a follow up

Reply:
updated